environment.yml

```diff
@@ -27,6 +27,6 @@ dependencies:
   - ska=1.0
   - sphinx=2.3.1
   - pip:
-    - git+https://github.com/czbiohub/bam2fasta.git@master#egg=bam2fasta
+    - git+https://github.com/czbiohub/bam2fasta.git@master#egg=bam2fasta==1.0.1dev0
```

Comment:
when you are rebuilding and want to use the most recent bam2fasta please change this not to pin down on a version